stable/rookout/Chart.yaml

```
@@ -1,10 +1,10 @@
 apiVersion: v1
-appVersion: "1.0"
+appVersion: "0.2.3"
```

```suggestion
appVersion: "0.3.9"
```
which is the image tag?